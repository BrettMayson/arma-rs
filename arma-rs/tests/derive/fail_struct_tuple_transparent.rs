use arma_rs::{FromArma, IntoArma};

#[derive(FromArma, IntoArma)]
#[arma(transparent)]
struct DeriveTest(String, u32);

fn main() {}
