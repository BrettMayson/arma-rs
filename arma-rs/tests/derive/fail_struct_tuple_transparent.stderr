error: #[arma(transparent)] cannot be used on tuple like structs
 --> tests/derive/fail_struct_tuple_transparent.rs:4:8
  |
4 | #[arma(transparent)]
  |        ^^^^^^^^^^^
